### ISSUE TITLE
Label.text does not accept bytes

### ISSUE THREAD

**Issue by dessant** [kivy/kivy#4255]
We were discussing with @akshayaurora that TextInput already accepts both and it would be nice if Label would do the same. Or maybe not? Thoughts? :stuck_out_tongue:

**Comment by matham** [comment]
In py3 I don't think it should accept bytes. In py2 I assume it currently accepts both bytes and unicode. Or, we need to allow the user to specify encoding then we could accept bytes.

**Comment by SepehrHml** [comment]
i guess i can work on this one if you define what should be done :P .

**Comment by KeyWeeUsr** [comment]
Isn't unicode already used as core for TextInput and if you put into `text` bytes it only converts them? (I work with py2) If I'm right, then this should be the same for Label i.e. unicode as core and if someone wants to get bytes to/from that, then `.encode()` or `.decode()` is the right way for output, isn't it? Kivy could only convert to the core text type if the bytes are used and don't bother with them anymore.

**Comment by saqib1707** [comment]
@dessant  is this feature of label accepting both text and bytes still has to be added or you people dropped the idea ?

**Comment by tito** [comment]
You're right, bytes should not be showed, as we don't know the encoding. If you are with Python 3, then pass unicode string for all text in Kivy. Don't pass bytes :)
(same kind of enforcing that we have using Python 3 in many things)
